other/materials_designer/specific_examples/defect_point_adatom_island_mos2_pt.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    ]
</pre><p>We should explain how we came up with these coordinates, so that readers can follow the line of thought</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>